src/options/quantifiers_options.toml

```diff
@@ -200,6 +200,23 @@ name   = "Quantifiers"
   default    = "false"
   help       = "use fast enumeration to augment instantiations from MBQI"
 
+
+[[option]]
+  name       = "mbqiFastSygusExtVarsGrammar"
+  category   = "expert"
+  long       = "mbqi-fast-sygus-ext-vars-grammar"
+  type       = "bool"
+  default    = "false"
```

Comment:
Can you make this default "true" to preserve current behavior?  Ditto for `mbqiFastSygusFreeSymsGrammar`

Reply:
done:)